--- a/routes/login.js
+++ b/routes/login.js
@@ -1,5 +1,8 @@
 module.exports = function login () {
   return (req, res, next) => {
-    models.sequelize.query(`SELECT * FROM Users WHERE email = '${req.body.email}' AND password = '${hash}'`)
+    models.sequelize.query(
+      'SELECT * FROM Users WHERE email = ? AND password = ?',
+      { replacements: [req.body.email, hash], type: models.sequelize.QueryTypes.SELECT }
+    )
   }
 }
